morse_talk/__init__.py

```diff
@@ -1,2 +1,8 @@
 from .encoding import *
 from .decoding import *
+
+# Release data
+from morse_talk import release
```

Comment:
It's merely about the convention. According to PEP8
1. Relative imports are highly discouraged.
2. Wildcard import of methods isn't a good choice for not-so-large number of methods.

L1 and L2 of this files should also be changed.

For backwards compatibility, we should also use
```
from __future__ import absolute_import
```